api/src/main/java/jakarta/data/repository/BasicRepository.java

```diff
@@ -131,25 +131,25 @@ public interface BasicRepository<T, K> extends DataRepository<T, K> {
     /**
      * Retrieves an entity by its Id.
      *
-     * @param id must not be {@literal null}.
-     * @return the entity with the given Id or {@literal Optional#empty()} if none found.
+     * @param id must not be {@code null}.
+     * @return the entity with the given Id or {@code Optional#empty()} if none found.
```

Comment:
Thus one was probably intended to be `@link`,
```suggestion
     * @return the entity with the given Id or {@link Optional#empty()} if none is found.
```